### ISSUE TITLE
Swift Package Manager Support

### ISSUE THREAD

**Issue by twodayslate** [NeoTeo/SwiftGMP#10]
Add support for [Swift Package Manager](https://github.com/apple/swift-package-manager/blob/master/Documentation/Usage.md)

Previously discussed [here](https://github.com/NeoTeo/SwiftGMP/issues/6#issuecomment-243043354).

**Comment by NeoTeo** [comment]
Sorry for the late reply. [SPM does not yet support iOS ](https://github.com/apple/swift-package-manager) and so is not yet suitable for using for this.

**Comment by decanus** [comment]
@NeoTeo is there any way to support both swiftpackage manager and others as just merged in the VarInt repository

**Comment by NeoTeo** [comment]
I only intend to support the Swift Package Manager now that iOS support is coming.

**Comment by decanus** [comment]
Was support announced @NeoTeo?

**Comment by NeoTeo** [comment]
Yes, the SPM doc now states "Xcode 11 integrates with libSwiftPM to provide support for iOS, watchOS, and tvOS platforms."

**Comment by simonmcl** [comment]
@NeoTeo Any update on SPM support? I'm trying to add IPFS support to the package i'm building. I'm trying to use: https://github.com/ipfs-shipyard/swift-ipfs-http-client in my swift package, building it with carthage and include the outputs in my package is failing because I can't seem to build it on an M1 mac:

```
.....Projects/swift-ipfs-http-client/Carthage/Checkouts/SwiftGMP/GMPLib/iphonesimulator/libGMP.a(abs.o), building for iOS Simulator, but linking in object file built for iOS, for architecture arm64
```

I can't find any good solution to using IPFS in iOS, any info would be helpful
